### PR TITLE
Fix golint errors on controller pkg directory

### PR DESCRIPTION
**Pull request by j-griffith**
Fix golint errors in controller pkg (replaces previous work in smaller chunks)

### CODE REVIEW

File: pkg/controller/datavolume-controller.go [hunk 1]
```diff
@@ -43,19 +43,24 @@ import (
 	cdischeme "kubevirt.io/containerized-data-importer/pkg/client/clientset/versioned/scheme"
 	informers "kubevirt.io/containerized-data-importer/pkg/client/informers/externalversions/datavolumecontroller/v1alpha1"
 	listers "kubevirt.io/containerized-data-importer/pkg/client/listers/datavolumecontroller/v1alpha1"
-	. "kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/common"
```
Comment (rollandf) [inline pkg/controller/datavolume-controller.go]: In clone-controller.go, we import as "cdi".
Now we have both :
glog.V(common.Vdebug)
glog.V(cdi.Vdebug)

Comment (j-griffith) [reply pkg/controller/datavolume-controller.go]: Ahh, yeah thanks for noticing that; sorry about that; I'll update this in the morning and make it consistent and just use "common".  Sound ok?

Comment (rollandf) [reply pkg/controller/datavolume-controller.go]: Great! Thanks.

File: pkg/controller/clone-controller.go [hunk 2]
```diff
@@ -16,21 +16,26 @@ import (
...
+	//AnnCloneRequest sets our expected annotation for a CloneRequest
 	AnnCloneRequest = "k8s.io/CloneRequest"
-	AnnCloneOf      = "k8s.io/CloneOf"
-	// cloner pods annotations
-	AnnCloningCreatedBy   = "cdi.kubevirt.io/storage.cloningCreatedByController"
-	AnnClonePodPhase      = "cdi.kubevirt.io/storage.clone.pod.phase"
-	CloneUniqueID         = "cdi.kubevirt.io/storage.clone.cloneUniqeId"
+	// AnnCloneOf sets our expected annotation to inidcate where a clone originated from
+	AnnCloneOf = "k8s.io/CloneOf"
+	// AnnCloningCreatedBy provides an annotation to indicate a PVC was created by our clone controller
```
Comment (jeffvance) [inline pkg/controller/clone-controller.go]: not specific to clones. This anno indicates that a cdi controller created the object. For the importer this anno shows that the importer pod was created by the cdi controller.

File: pkg/controller/clone-controller.go [hunk 3]
```diff
@@ -16,21 +16,26 @@ import (
...
+	AnnCloningCreatedBy = "cdi.kubevirt.io/storage.cloningCreatedByController"
+	// AnnClonePodPhase provides an annotation to indicate the pod phase of the our clone pod
```
Comment (jeffvance) [inline pkg/controller/clone-controller.go]: again, not specific to the cloner. To see the full scope of use try `grep -rn  <Ann-in-question> pkg`

**Comment by jeffvance** [comment]
Just a couple of comments but looks good.
Needs rebase and consider your newer prs and decide if you want to roll this into one or more of them. Otherwise can merge this after response to comments.

File: pkg/controller/clone-controller.go (hunk 2 above)
Comment (j-griffith) [reply pkg/controller/clone-controller.go]: Cool, thanks!  Got carried away and didn't check closely.

File: pkg/controller/clone-controller.go (hunk 3 above)
Comment (j-griffith) [reply pkg/controller/clone-controller.go]: ack, update on the way

**Comment by jeffvance** [comment]
lgtm

**Comment by jeffvance** [comment]
Needs manual rebase.

**Comment by awels** [comment]
@j-griffith The travis error seems legit, maybe an order of merging patches is needed?

**Comment by jeffvance** [comment]
build error:
```
../../pkg/controller/util.go:280:11: undefined: common.KeyAccess
../../pkg/controller/util.go:290:11: undefined: common.KeySecret
```

**Comment by j-griffith** [comment]
Yep, I'm working on fixing it now; the merge went sideways.

**Comment by j-griffith** [comment]
Grrrrr, one more (I think)

**Pull request merged**